packages/core/src/facade/f-blob.ts

```diff
@@ -0,0 +1,197 @@
+     * @returns a blob
+     * @example
+     * ```ts
+     * const blob = UniverApi.newBlob(blob);
+     * const newBlob = blob.getBlob();
+     * ```
+     */
+    getAs(contentType: string) {
+        const newBlob = this.copyBlob();
+        newBlob.setContentType(contentType);
+        return newBlob;
+    }
+
+    /**
+     * get the blob as a string
+     * @returns a string
+     * @example
+     * ```ts
+     * const blob = UniverApi.newBlob(blob);
+     * const newBlob = blob.getDataAsString();
+     * console.log(newBlob);
+     * ```
+     */
+    getDataAsString(): Promise<string>;
+    /**
+     * get the blob as a string
+     * @param charset the charset
+     * @returns a string
+     * @example
+     * ```ts
+     * const blob = UniverApi.newBlob(blob);
+     * const newBlob = blob.getDataAsString('iso-8859-1');
+     * console.log(newBlob);
+     * ```
+     */
+    getDataAsString(charset?: string): Promise<string>;
+    getDataAsString(charset?: string): Promise<string> {
+        if (this._blob === null) {
+            return Promise.resolve('');
+        }
+
+        if (charset === undefined) {
+            return this._blob!.text();
+        }
+        return new Promise((resolve, reject) => {
+            this._blob!.arrayBuffer().then((arrayBuffer) => {
+                const text = new TextDecoder(charset).decode(arrayBuffer); // 解码为字符串
+                resolve(text);
+            }).catch((error) => {
+                reject(new Error(`Failed to read Blob as ArrayBuffer: ${error.message}`));
+            });
+        });
+    }
+
+    /**
+     * Gets the data stored in this blob.
+     * @returns a byte array
+     * @example
+     * ```ts
+     * const blob = UniverApi.newBlob(blob);
+     * const newBlob = blob.getBytes();
+     * console.log(newBlob);
+     * ```
+     */
+    getBytes(): Promise<Uint8Array> {
+        if (!this._blob) {
+            return Promise.reject(new Error('Blob is undefined or null.'));
+        }
+        return this._blob.arrayBuffer().then((buffer) => new Uint8Array(buffer));
+    }
+
+    /**
+     * Sets the data stored in this blob.
+     * @param bytes a byte array
+     * @example
+     * ```ts
+     * const blob = UniverApi.newBlob();
+     * const bytes = new Uint8Array(10);
+     * blob.setBytes(bytes);
+     * ```
+     */
+    setBytes(bytes: Uint8Array): void {
+        this._blob = new Blob([bytes]);
+    }
+
+    /**
+     * Sets the data stored in this blob.
+     * @param data a string
+     * @example
+     * ```ts
+     * const blob = UniverApi.newBlob();
+     * blob.setDataFromString('Hello, World!');
+     * ```
+     */
+    setDataFromString(data: string): void;
+    /**
+     * Sets the data stored in this blob.
+     * @param data a string
+     * @param contentType the content type refer to https://developer.mozilla.org/en-US/docs/Web/HTTP/MIME_types/Common_types
```

Comment:
the optional param should comment like following:
@param {string} [contentType] - The contentType, which refer to  https://developer.mozilla.org/en-US/docs/Web/HTTP/MIME_types/Common_types